Over21/controllers/ViewController.swift

```diff
@@ -81,6 +85,11 @@ class ViewController: UIViewController {
         setupCapture()
         
         setupUIElements()
+        initUPOS()
+        btLookup()
+        
+        //parseDecodedBarCode("C1USA0427797024MSC1380273242\n<<6010014M2405193IRL<<<<<<<<<<<4MILLS<<KEVIN<JAMES<<<<<<<<<<<<")
+        //checkIfUserIsOver21()
```

Comment:
This commented out line should be removed

Reply:
removed this comment line